services/web/server/src/simcore_service_webserver/projects/projects_db.py

```diff
@@ -462,6 +462,15 @@ async def update_project_without_checking_permissions(
             )
             return result.rowcount == 1
 
+    async def update_project_last_change_timestamp(self, project_uuid: ProjectIDStr):
+        async with self.engine.acquire() as conn:
+            await conn.execute(
+                # pylint: disable=no-value-for-parameter
+                projects.update()
+                .values(last_change_date=now_str())
+                .where(projects.c.uuid == project_uuid)
+            )
+
```

Comment:
Question: How does this behave if the project is missing? Will it raise an error?

Reply:
Hey, good point thank you! no, the current behavior will not raise an error, I will add a check for that so the error is raised.